gnd/src/main/java/com/google/android/gnd/system/CameraManager.java

```diff
@@ -97,4 +106,15 @@ private Completable sendCapturePhotoIntent() {
           }
         });
   }
+
+  public Completable addPhotoToGallery(String photoPath) {
+    return Completable.fromRunnable(
+        () -> {
+          Intent mediaScanIntent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE);
+          File f = new File(photoPath);
```

Comment:
Please use `file` instead of `f` or make it inline.

Reply:
Done, ty!